app/views/users/show_follow.html.erb

```diff
@@ -64,7 +64,7 @@
     </div>
     </br>
     <% if @pagy %> 
```

Comment:
@KarishmaVanwari Instead of adding the condition inside, we could just use the general if statement on line 66. What do you think?

Reply:
@TildaDares I've pushed a commit that now uses general if statement. Hope it works!